cas-server-support-jpa-ticket-registry/src/main/java/org/apereo/cas/config/JpaTicketRegistryConfiguration.java

```diff
@@ -47,6 +47,7 @@
     public String[] ticketPackagesToScan() {
         return new String[] {
                 "org.apereo.cas.ticket",
+                "org.apereo.cas.util.jpa",
```

Comment:
And which database is that? And what field exactly are you referring to? the creation-time or the expiration policy?